app/components/Send/SendPanel/index.jsx

```diff
@@ -63,9 +65,11 @@ const SendPanel = ({
   handleAddPriorityFee,
   fees,
   address,
-  maxNumberOfRecipients
+  maxNumberOfRecipients,
+  showSendModal,
+  pushQRCodeData
 }: Props) => {
-  function shouldDisableSendButton(sendRowDetails) {
+  const shouldDisableSendButton = sendRowDetails => {
```

Comment:
sendRowDetails should have a type here

Reply:
This passes flow validation and actually wasn't even a part of the functionality implemented within this PR. I'm happy to add it, but for some reason flow was barking at me for adding types to a function defined within a stateless component.